topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+
+> ### {% icon details %} FASTQ sequence quality format
+>
+> FASTQ format is a text-based format for storing both a biological sequence and its corresponding quality scores. Both the sequence letter and the quality score are encoded with a single ASCII character for brevity. You can find more info in the [Wikipedia article](FASTQ format).
+>
+{: .details}
+
+## Quality control using FastQC and MultiQC
+
+__FastQC__ is one of the most widely used tools to __check the quality__ of the samples generated by High Throughput Sequencing (HTS) technologies. 
+
+> ### {% icon hands_on %} Hands-on: Quality check
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `soil collection` 
+>
+> 2. Rename the outputs as `FastQC unprocessed: Raw` and `FastQC unprocessed: Web` 
+> 3. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*:
+>
+>		- `FastQC unprocessed: Raw`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+> 
+>
+{: .hands_on}
+
+__FastQC__ provides information on various parameters, such as the range of quality values across all bases at each position. __MultiQC__ allows summarizing the output of different outputs from __FastQC__.
+
+![FastQC plots length](../../images/metagenomics-nanopore/fastqc_sequence_length_distribution_plot.png "Sequence length distribution")
+
+In the first place, we are going to analyse the sequence length distribution of the different datasets.
+
+> ### {% icon question %} Question
+>
+> 1. Can you explain the sequence length distribution plot?
+>
+> > ### {% icon solution %} Solution
+> >
+> > The main peak, around 1700 bp, corresponds approximately to the length of the gene coding for 16S rRNA. There is also a secondary peak around 200 bp, which may be due to __truncated amplifications__, or as a result of __non-specific hybridization of primers__ used for PCR.
+> >
+> {: .solution}
+{: .question}
+
+
+![FastQC plots quality](../../images/metagenomics-nanopore/fastqc_per_base_sequence_quality_plot.png "Per base sequence quality")
+
+
+
+If we examine figure 3, we can see that up to 3000 bp the quality of our sequencing data is around a __Phred score__ of 12, which is a relatively low value compared to other sequencing technologies. This is explained because Nanopore reads poses __high error rates in the basecalled reads__ (10% as compared to 1% for Illumina). However, Nanopore sequencing generates __very long reads__ (in theory only limited by the mechanisms of extraction of the genetic material), enabling the sequencing of the complete 16S rRNA gene, which makes it possible to identify bacterial taxa at higher resolution.
```

Comment:
```suggestion
If we examine figure 4, we can see that up to 3000 bp the quality of our sequencing data is around a __Phred score__ of 12, which is a relatively low value compared to other sequencing technologies. This is explained because Nanopore reads poses __high error rates in the basecalled reads__ (10% as compared to 1% for Illumina). However, Nanopore sequencing generates __very long reads__ (in theory only limited by the mechanisms of extraction of the genetic material), enabling the sequencing of the complete 16S rRNA gene, which makes it possible to identify bacterial taxa at higher resolution.
```